Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6163,22 +6163,35 @@ def com_google_fonts_check_metadata_category_hint(family_metadata):
     proposal = 'https://github.com/googlefonts/fontbakery/issues/3886'
 )
 def com_google_fonts_check_colorfont_tables(ttFont):
-    """Fonts must have neither or both the tables 'COLR' and 'SVG'."""
+    """Check font has correct color font tables"""
     SUGGESTED_FIX = ("To fix this, please run the font through the maximum_color tool"
                      " that installs as part of the nanoemoji package"
                      " (https://github.com/googlefonts/nanoemoji)")
-    if 'COLR' in ttFont.keys() and 'SVG' not in ttFont.keys():
-        yield FAIL,\
-              Message('missing-table',
-                      "This is a color font (it has a 'COLR' table)"
-                      " but it lacks an 'SVG' table. " + SUGGESTED_FIX)
-    elif 'COLR' not in ttFont.keys() and 'SVG' in ttFont.keys():
-        yield FAIL,\
-              Message('missing-table',
-                      "This is a color font (it has a 'SVG' table)"
-                      " but it lacks an 'COLR' table. " + SUGGESTED_FIX)
-    else:
-        yield PASS, "Looks good!"
+    if "COLR" in ttFont:
+        colr_table = ttFont["COLR"]
+        if colr_table.version == 0 and "SVG" in ttFont:
+            yield FAIL, Message(
```

Comment:
FAIL is a bit too harsh in the case of COLRv0 + SVG. The latter may still be useful for some applications or old browser version. I suggest a WARNING instead

Reply:
I opted for FAIL since Cairo Play was reverted, https://github.com/google/fonts/pull/5135

Reply:
Though, now I see that in Rod's comment which Felipe pasted in https://github.com/googlefonts/fontbakery/issues/3888 it says it explicitly says

> Any other state is invalid, in particular COLRv0 + SVG should _fail_.

Maybe then keep it a FAIL then

Reply:
Ok cool.

Reply:
Yeah, ok. In the end this is the googlefonts profile, not the general one, so we make our own rules